shared/test/test_net_sim_api.rb

```diff
@@ -0,0 +1,55 @@
+require 'minitest/autorun'
+require 'rack/test'
+require File.expand_path '../../../deployment', __FILE__
+require File.expand_path '../../middleware/net_sim_api', __FILE__
+
+ENV['RACK_ENV'] = 'test'
+
+class NetSimApiTest < Minitest::Unit::TestCase
+
+  def create_record(record)
+    @net_sim_api.post "/v3/netsim/#{@shard_id}/#{@table_name}", record.to_json, 'CONTENT_TYPE' => 'application/json;charset=utf-8'
```

Comment:
No such helper exists right now, but I can point you to every spot in the code that currently knows how these URLs are constructed.

* On the server
  * net_sim_api.rb which defines the routes.
  * test_net_sim_api.rb which tests the routes.
* On the client
  * [NetSimTable.js](https://github.com/code-dot-org/code-dot-org/pull/3134/files#diff-b4e0cb76b01a931f585c5352404e52afR51) which defines the API base URL
  * [clientApi.js](https://github.com/code-dot-org/code-dot-org/blob/staging/shared/js/clientApi.js) which is still used because, other than the base URL, the NetSim API is still the same as the channels API.  Eventually we may have to break this out too.

Right now I think the only place we could extract some helpers and actually see them used would be for net_sim_api.rb to expose some URL-constructing helpers and use them in the tests - but even then, they would only have one usage since net_sim_api.rb itself uses route regexps.  This seems like a premature optimization.